spyder/app/mainwindow.py

```diff
@@ -264,6 +264,11 @@ def setup_logging(cli_options):
                             filemode='w+')
 
 
+QDARKSTYLE_REQVER = '>=2.6.4'

```

Comment:
@juanis2112, I forgot to tell you, but please you add a block comment above this line, like this:

```
#==============================================================================
# Dependencies
#==============================================================================
```

Thanks!

Reply:
To comply with PEP 8 and be consistent with elsewhere, it should be like this:

```python
# =============================================================================
# Dependencies
# =============================================================================
```

Reply:
Yeah, right.